angel_castro/ejercicio.md

```diff
@@ -0,0 +1,31 @@
+```
+Sub evento_escolar()
+     dtr = 0
+     si_abonaron = 0
+     no_abonaron = 0
+     abn_s = 0
+     While dtr <= 3000000
+         dnr_r = Int(InputBox("cuanto va a donar"))
+         If dnr_r > 0 Then
+            si_abonaron = si_abonaron + 1
+            dtr = dtr + dnr_r
+            If dnr_r >= 10000 Then
```

Comment:
@angelcastro0923 You have to make the change of variable in the flowchart, you put a variable that does not exist "a"